mkdocs/docs/getting-started/your-first-experiment.md

```diff
@@ -4,57 +4,65 @@ template: main.html
 
 # Your First Experiment
 
-!!! tip "Load test https://example.com and validate SLOs" 
-    Use an [Iter8 experiment](concepts.md#what-is-an-iter8-experiment) to load test https://example.com and validate latency and error-related service level objectives (SLOs).
+!!! tip "Load test an HTTP Service and validate SLOs" 
+    Use an [Iter8 experiment](concepts.md#what-is-an-iter8-experiment) to load test an HTTP service (https://example.com) and validate latency and error-related [service level objectives (SLOs)](../user-guide/topics/slos.md).
 
 ## 1. Install Iter8
 Install Iter8 using [Go 1.16+](https://golang.org/) as follows.
 ```shell
 go install github.com/iter8-tools/iter8@latest
-# you can now run iter8 (from your gopath bin/ directory)
 ```
+You can now run `iter8` (from your gopath bin/ directory)
 
-## 2. Download experiment
-Download the `load-test` experiment folder from [Iter8 hub](../user-guide/topics/iter8hub.md) as follows.
+## 2. Download experiment chart
+Download the `load-test` [experiment chart](concepts.md#experiment-chart) from [Iter8 hub](../user-guide/topics/iter8hub.md) as follows.
 
 ```shell
 iter8 hub -e load-test
 ```
+This creates a local folder called `load-test` containing the chart.
 
-## 3. Run experiment
-[Iter8 experiments](concepts.md#what-is-an-iter8-experiment) are specified using the `experiment.yaml` file. The `iter8 run` command reads this file, runs the specified experiment, and writes the results of the experiment into the `result.yaml` file.
-
-Run the experiment you downloaded above as follows.
-
+## 3. Generate `experiment.yaml`
+Generate the `experiment.yaml` file which specifies your load test experiment.
 ```shell
 cd load-test
-iter8 run
+iter8 gen exp --set url=https://example.com
```

Comment:
We can still start with an `experiment.yaml`. Is using `iter8 gen exp` as part of the first experiment maybe more complex than it needs to be? I agree that the chart is flexible. But it seems to suggest that `iter8 gen exp` is always needed. Is this one more concept that we don't need to start?

Reply:
If *every* experiment is a helm chart, then perhaps there is no need for the `gen` command; just make `run` do the generation.

Reply:
I would like the experiment chart to be flexible enough so that most users don't ever need to understand or look into `experiment.yaml` or tasks or any such thing... but simply reuse with different values.
***
Definitely more documentation is needed for supported values, which I plan to include before the blog writeup.
***
So, yes, the concept we want to push is the templated experiment, not the experiment.yaml.

Reply:
Yes, we can make `run` to the generation directly (we will explore this option shortly).
***
At the moment, `gen k8s` seems to require `experiment.yaml`. It doesn't seem right to `run` locally first just for the same of the `experiment.yaml`.

Reply:
We can eventually consider your suggestion captured in this issue now: https://github.com/iter8-tools/iter8/issues/1108

Reply:
> At the moment, gen k8s seems to require experiment.yaml. It doesn't seem right to run locally first just for the sake of generating the experiment.yaml.

Agreed. `gen k8s` would have to change as well, I think.